cachalot/utils.py

```diff
@@ -95,9 +95,24 @@ def get_table_cache_key(db_alias, table):
     return sha1(cache_key.encode('utf-8')).hexdigest()
 
 
-def _get_tables_from_sql(connection, lowercased_sql):
-    return {t for t in connection.introspection.django_table_names()
-            + cachalot_settings.CACHALOT_ADDITIONAL_TABLES if t in lowercased_sql}
+def _get_tables_from_sql(connection, lowercased_sql,
+                         enable_quote=False):
+    """Returns names of involved tables after analyzing the final SQL query."""
+    return {table for table in connection.introspection.django_table_names()
+            + cachalot_settings.CACHALOT_ADDITIONAL_TABLES
+            if _quote_table_name(table, connection, enable_quote) in lowercased_sql}
```

Comment:
nice addition 👍 

 I also wanna double check that `quote_name` is not making a call to the database since that sometimes happens for some obscure thing.

Reply:
> nice addition +1
> 
> I also wanna double check that `quote_name` is not making a call to the database since that sometimes happens for some obscure thing.

At first glance `quote_name` seems to be safe, but it’s better to double-check of course!
https://github.com/django/django/blob/e703b152c6148ddda1b072a4353e9a41dca87f90/django/db/backends/mysql/operations.py#L177